add configurable picture path in preprare for work (#88)
opengisch/QField-docs: en/prepare/picture_path.rst

Configurable picture path
=========================

> QFieldSync gives the possibility to configure the path of the attachments.

Reviewer: ```suggestion
QFieldSync gives the possibility to configure the path of picture attachments.
```

Let's be consistent in here (picture, photo, attachments - all refers to the same)

1. Go to QFieldSync plugin --> Project configuration
2. Select the table "Photo Naming"

> 3. Choose the layer, the field and introduce the expression 

Reviewer: ```suggestion
3. Choose the layer, the field and configure the expression 
```

> Use expressions to configure the path of the attachments. Usually DCIM is used as the main folder and it's the default.

Reviewer: ```suggestion
Use expressions to configure the path of the attachments. By default, pictures are saved into the `DCIM` folder with a timestamp as name.
```
